astlib/dune

```diff
@@ -2,7 +2,7 @@
  (name astlib)
  (public_name ppxlib.astlib)
  (libraries ocaml-compiler-libs.common compiler-libs.common)
- (flags -w -9)
+ (flags -w -9 -I +dynlink)
```

Comment:
This doesn't look right for a Dune-based system - the fix ought to be adding `dynlink` to a `libraries` field somewhere, but I also didn't see this warning triggering locally?

Reply:
Good catch, thanks! Fun side-note: I hadn't even added the `dunlink` thing to the right dune file.

Reply:
Ah, I was wondering how it had ended up there 🙂